### PR TITLE
Update Ukrainian translation

### PR DESCRIPTION
**Pull request by sakhnik**
Here's the Ukrainian translation update. Rest in peace, Bram.

### CODE REVIEW

File: src/po/uk.cp1251.po [hunk 1]
```diff
@@ -12,18 +12,18 @@
...
-"POT-Creation-Date: 2022-05-02 17:40+0300\n"
-"PO-Revision-Date: 2022-05-20 17:12+0300\n"
+"POT-Creation-Date: 2023-08-24 11:13+0300\n"
+"PO-Revision-Date: 2023-08-25 13:12+0300\n"
 "Last-Translator: ������� ������ <sakhnik@gmail.com>\n"
 "Language-Team: Ukrainian\n"
 "Language: uk\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=cp1251\n"
+"Content-Type: text/plain; charset=UTF-8\n"
```
Comment (k-takata) [inline src/po/uk.cp1251.po]: This looks wrong.
`uk.cp1251.po` should be automatically updated by running `make` in the `src/po/` directory.

Comment (sakhnik) [reply src/po/uk.cp1251.po]: It didn't, by the way. But I've updated it manually. Thanks!

File: src/po/uk.cp1251.po [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #
 # Ukrainian translation for Vim
 #
-# Generated from uk.po, DO NOT EDIT
+# Original translations
```
Comment (k-takata) [inline src/po/uk.cp1251.po]: This looks also wrong.
This should not happen if you run `make`.

Comment (sakhnik) [reply src/po/uk.cp1251.po]: Sorry

File: src/po/uk.cp1251.po (hunk 1 above)
Comment (k-takata) [reply src/po/uk.cp1251.po]: > It didn't, by the way.

Hmm. If `uk.cp1251.po` is newer than `uk.po`, `make` will not update it.
I'm not sure if this is the case, though.

Comment (sakhnik) [reply src/po/uk.cp1251.po]: All right, that was my manual intervention that messed things up. When I just edit uk.po and leave uk.cp1251.po alone, everything works fine. I just need to remember this.

**Comment by sakhnik** [comment]
Thanks, @chrisbra. I'm still refining the messages while updating the translation of Neovim. So I'm going to follow up with another pull request soon.

**Pull request merged**